src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -122,103 +135,111 @@ export const WorkspaceMenu = ({ coreStart, registeredUseCases$ }: Props) => {
           {currentWorkspace ? (
             <>
               <EuiFlexItem grow={false}>
-                <EuiAvatar
-                  size="m"
-                  type="space"
-                  name={currentWorkspaceName}
-                  color={getValidWorkspaceColor(currentWorkspace?.color)}
-                  initialsLength={2}
-                />
+                <EuiIcon size="xl" color="success" type="wsObservability" />
```

Comment:
Should we use different icon type based on use case of current workspace?

Reply:
yes, based on the UX design, if it's inside a workspace, the icon type changed to wsObservability. in the opposite case, it's wsSelector.

Reply:
I suppose that's just a demo for specific case when in workspace, we should display based on the use case of current workspace, @SuZhou-Joe  am I right?

Reply:
oh, I understand, will updated.

Reply:
I suppose that's just a demo for specific case when in workspace, we should display different icon type based on the use case of current workspace, @SuZhou-Joe right?